Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -29,44 +30,38 @@ type IndicatorSize = number | 'small' | 'large';
 
 type IOSProps = $ReadOnly<{|
   /**
-   * Whether the indicator should hide when not animating (true by default).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#hideswhenstopped
-   */
+    Whether the indicator should hide when not animating.
+
+    @platform ios
+  */
   hidesWhenStopped?: ?boolean,
 |}>;
 type Props = $ReadOnly<{|
   ...ViewProps,
   ...IOSProps,
 
   /**
-   * Whether to show the indicator (true, the default) or hide it (false).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#animating
+   	Whether to show the indicator (`true`) or hide it (`false`).
    */
   animating?: ?boolean,
 
   /**
-   * The foreground color of the spinner (default is gray).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#color
-   */
+    The foreground color of the spinner.
+
+    @default {@platform android} `null` (system accent default color)
+    @default {@platform ios} '#999999'
+  */
   color?: ?ColorValue,
 
   /**
-   * Size of the indicator (default is 'small').
-   * Passing a number to the size prop is only supported on Android.
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#size
-   */
+    Size of the indicator.
+
+    @type enum(`'small'`, `'large'`)
```

Comment:
**Long term comment - no need to fix before merging**

I'm not sure we should be embedding Markdown backticks in types like this. This may be fine to ship as a starting point, but later we should consider migrating to a type syntax with no manual formatting. This type of formatting should "just work" implicitly, like syntax highlighting.

```suggestion
    @type enum('small', 'large')
```

Reply:
Yes, this needs to be addressed. To fix that we need to split and add the script for formating the values of enums. We can try that out to fix.